lms/startup.py

```diff
@@ -58,7 +58,7 @@ def add_mimetypes():
     mimetypes.add_type('application/font-woff', '.woff')
 
 
-def enable_theme():
+def enable_stanford_theme():
```

Comment:
I agree with @antoviaque here. What’s the plan? Deprecate the old theming mechanism in favor of comprehensive theming?

Then why not call it `enable_deprecated_theme`?